moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1644,6 +1644,11 @@ void MoveGroupInterface::setRandomTarget()
 }
 
 const std::vector<std::string>& MoveGroupInterface::getJointNames() const
+{
+  return getVariableNames();
+}
```

Comment:
Please move this implementation to the header - for inlining (and documenting use of the new function).

Reply:
I put it here on purpose to keep the symbol in the built library during deprecation and it's documented in the deprecation tag already, but I don't care too much and will change it.

Reply:
I see, that's a good argument too. Decide yourself. More importantly: this PR seems to break on Melodic (`master-ci`)

Reply:
Yes, live without f-strings is hard. :) Should be fixed now.

Reply:
This type of format string is available in python 2.7:
```
a=1
"a={}".format(a)
```
You just don't yet have `f"{a}"`. Are you sure that this was causing the CI failure? I didn't have a look myself, sorry.

Reply:
That's just how I addressed it. :)
Yes, the CI output for failing tests is horrible, but that was the underlying mistake.